src/main/kotlin/com/manimdsl/linearrepresentation/Objects.kt

```diff
@@ -127,11 +127,14 @@ data class VariableBlock(
 sealed class DataStructureMObject(
     open val type: DataStructureType,
     open val ident: String,
+    open val uid: String,
```

Comment:
What is the `uid`?

Reply:
it's the string users use to define positioning for a specific variable - e.g. "x", "func.x", "stack1", we use that to find everything for the variable/data structure